apps/ima/index.js

```diff
@@ -31,11 +31,167 @@ module.exports = {
+      behaviours: SaveFormSession,
+      fields: [
+        'legal-representative-fullname',
+        'legal-representative-organisation',
+        'legal-representative-house-number',
+        'legal-representative-street',
+        'legal-representative-townOrCity',
+        'legal-representative-county',
+        'legal-representative-postcode',
+        'legal-representative-phone-number',
+        'is-legal-representative-email',
+        'legal-representative-email'
+      ],
+      continueOnEdit: true,
+      locals: { showSaveAndExit: true },
+      next: '/your-location'
+    },
+    '/helper-details': {
+      behaviours: SaveFormSession,
+      fields: [
+        'someone-else-fullname',
+        'someone-else-relationship',
+        'someone-else-organisation'
+      ],
+      continueOnEdit: true,
+      locals: { showSaveAndExit: true },
+      next: '/your-location'
+    },
+    '/your-location': {
+      behaviours: SaveFormSession,
+      forks: [
+        {
+          target: '/full-name',
+          condition: {
+            field: 'in-the-uk',
+            value: 'yes'
+          }
+        },
+        {
+          target: '/cannot-use-form',
+          condition: {
+            field: 'in-the-uk',
+            value: 'no'
+          }
+        }
+      ],
+      fields: ['in-the-uk'],
+      locals: { showSaveAndExit: true },
+      continueOnEdit: true,
+      next: '/full-name'
+    },
+    '/full-name': {
+      behaviours: SaveFormSession,
+      fields: ['name'],
+      locals: { showSaveAndExit: true },
+      next: '/your-email'
+    },
+    '/your-email': {
+      behaviours: SaveFormSession,
+      fields: ['current-email'],
+      forks: [
+        {
+          target: '/phone-number',
+          condition: {
+            field: 'current-email',
+            value: 'yes'
+          }
+        },
+        {
+          target: '/email',
+          condition: {
+            field: 'current-email',
+            value: 'no'
+          }
+        }
+      ],
+      locals: { showSaveAndExit: true },
+      continueOnEdit: true,
+      next: '/phone-number',
+      backLink: 'full-name'
+    },
+    '/email': {
+      behaviours: SaveFormSession,
+      fields: ['email-address', 'email-address-details'],
+      locals: { showSaveAndExit: true },
+      next: '/phone-number'
+    },
+    '/phone-number': {
+      behaviours: SaveFormSession,
+      fields: ['phone-number', 'phone-number-details'],
+      locals: { showSaveAndExit: true },
+      next: '/immigration-detention'
+    },
+    '/immigration-detention': {
+      behaviours: SaveFormSession,
+      forks: [
+        {
+          target: '/', // TODO target url will be added by next TASK
+          condition: {
+            field: 'has-address',
+            value: 'yes'
+          }
+        },
+        {
+          target: '/medical-record',
+          condition: {
+            fields: 'has-address',
+            value: 'no'
+          }
+        }
+      ],
```

Comment:
Forks are the wrong way round (the 'yes' option should take user to the '/medical-record' page)

Reply:
The UCD keeps changing this requirment with notifying us, now the yes and no leads to same path

Reply:
done